File: crates/axhub-helpers/tests/fixtures/site-scan/go/bad.go
package sample

import (
	"database/sql"
	"net/http"
)

func load() {
	resp, _ := http.Get("https://api.axhub.dev/v1/posts")
	conn, _ := sql.Open("postgres", "dsn")
	url := "https://backend.example.com/data"
	_ = resp
	_ = conn
	_ = url
}

File: crates/axhub-helpers/tests/fixtures/site-scan/ruby/bad.rb
require "net/http"
require "pg"

def load
  res = Net::HTTP.get(URI("https://api.axhub.dev/v1/posts"))
  conn = PG.connect(dbname: "x")
  url = "https://backend.example.com/data"
  [res, conn, url]
end
